lib/drawers/head-components.vue

```diff
@@ -72,6 +72,9 @@
       openSettings(id) {
         const path = 'settings';
 
+        // If a different element is selected, the overlay form will use the
```

Comment:
I'd shorten this to something like "unselect to prevent overlay from using wrong schema"